.deployment.config.json

```diff
@@ -6,7 +6,7 @@
       "sequential": ["us-east-1"]
     },
     "environments_order": {
-      "sequential": ["dev", "qa", "prd"]
+      "sequential": ["dev", "prd"]
```

Comment:
no `stg`?

Reply:
Yes we should probably replace qa by staging instead !